### ISSUE TITLE
Feature: Custom fonts

### ISSUE THREAD

**Issue by nilsbyte** [Swordfish90/cool-retro-term#439]
It would be really nice to have the ability to use custom fonts installed on the system. I like to use CPMono_v07 as it's a very popular monospaced font.

**Comment by seguri** [comment]
I agree, I would like to see `powerlevel9k` working correctly. I know that people using a CRT terminal didn't have that theme available at the time, but well we are in 2018 :)

**Comment by Alexey104** [comment]
@seguri, i'm using powerlevel10k theme with cool-retro-term, and it works fine after adding 'export TERM=xterm-256color' to ~/.zshrc(or ~/.bashrc, depending on what you use). It doesn't work with all CRT fonts, but with Source Code Pro i don't have any problems:
![Screenshot from 2019-06-12 17-03-28](https://user-images.githubusercontent.com/38672094/59357742-0c8e9f00-8d34-11e9-98a6-74bdd739c69f.png)
![Screenshot from 2019-06-12 17-04-39](https://user-images.githubusercontent.com/38672094/59357885-4495e200-8d34-11e9-9e82-6ed4556376db.png)
![Screenshot from 2019-06-12 17-04-59](https://user-images.githubusercontent.com/38672094/59357887-465fa580-8d34-11e9-9316-072e974cbf25.png)

**Comment by Bug-Reaper** [comment]
Hey @alexey104 how are you setting a custom font, I don't see that option in the GUI settings?

**Comment by Alexey104** [comment]
@Bug-Reaper, oh, sorry, I don't use CRT anymore, and, honestly, I don't remember how I set it up. But I always use UbuntuMonoNerdFont for my terminal, and, if I remember correctly, you are able to choose this font in CRT preferences if you have it installed.
